SensorFusion_src/parse.c

```diff
@@ -1 +1,76 @@
-// the code for parsing the input file and converting it into a 1Xn array goes here
\ No newline at end of file
+/*******************************************************************************************/
+/* Author : Shyam Bhuptani 
+/* Function : Parse 
+/* Application : This function converts the input csv file into a sensor reading1-D array
+/* this parse file is for tag 1 which only works fine for one time value
+/* here the number of sensor is fixed which is 3
+/* Input : csv file of data  with 3 fields Time(24hr clock) , sensor_name, value
+/* Output : sensor reading array of size N (double)
+*/
+
+/*Including dependancies*/
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>	
+
+#include "../include/parse.h"	 
+//Number of sensors are fixed
+#define num_of_sensor 3
+
+/*Function declaration which returns a 2D array */
+double *Parse(char file_name[])
+{
+	printf("file name is %s\n",file_name);
+    int iterations = 0;
+    double *sensor_readings = (double *) malloc(num_of_sensor * sizeof(double *));
+	printf("here1");
+    FILE* stream = fopen(file_name,"r");
+	printf("here2");
+	/*If stream is not present*/
+    if (!stream){
+        printf("Can't Open file");
+    }
+    char line[1024];

```

Comment:
1024 should be defined with a define macro.

Reply:
Got it!!